### DISCUSSION TITLE
Possible to redirect url through serve function?

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#933] [Q&A] [answered]
Currently, I am using ```@on()``` to handle my hash routing. I have two screens, #1 and #2. I am serving at ```'/'``` however. Currently navigating to ```'/'``` will take the users to the #1 screen, but the route will remain at ```'/'```. I want to continue to serve to base, so that users don't have to worry about entering any routing, so I would ideally like to have my serve route to ```'/'``` but forward to ```'/#1'```. Is there any way to do this?

**Comment by Alex-Wenner-FHR** [comment]
I have tried to do things like ```if q.args['#'] == None``` (if we are at base url) ```q.args['#'] = '#1'``` but this doesn't seem to work!

**Comment by vopani** [comment, marked as answer]
You can use the `redirect` in `meta_card`: https://wave.h2o.ai/docs/examples/meta-redirect

eg: something like `q.page['meta'].redirect = '#1'`

**Reply by Alex-Wenner-FHR** [reply]
Ah this is awesome. I can't believe I overlooked that. Super helpful, thank you!
